src/access/acp_v1.ts

```diff
@@ -18,25 +18,18 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
-
+import { v4 as uuidv4 } from "uuid";
```

Comment:
We normally just write: `import { v4 } from "uuid";` but I think renaming here provides more clarity, so 👍